modules/internal/ChapelAutoAggregation.chpl

```diff
@@ -102,18 +103,27 @@ module ChapelAutoAggregation {
       const bufferSize = dstBuffSize;
       const myLocaleSpace = LocaleSpace;
       var opsUntilYield = yieldFrequency;
-      var lBuffers: [myLocaleSpace] [0..#bufferSize] aggType;
+      var lBuffers: c_ptr(c_ptr(aggType));
       var rBuffers: [myLocaleSpace] remoteBuffer(aggType);
```

Comment:
Why is this still not using `c_ptr`?

Reply:
I only switched things that are used in the `copy()` routine to be `c_ptr` since that's the routine that's called all the time. `rBuffers` is only used when we need to flush a buffer so the overhead of a chapel array vs c_ptr is negligible.